rhino-deinst

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 set -e
 
-if ! command -v dialog > /dev/null; then
```

Comment:
None of these are subshells. A subshell would be something like `( echo "hello" )`. Brackets `{}` are not subshells, and don't inherently slow down a program. On top of that, performance is not the goal of this program. It doesn't need to be lightning fast.

Reply:
I meant the `$( expr ... )` and `$( apt ...)` lines.